aes/src/ni/aes128/expand.rs

```diff
@@ -19,9 +19,9 @@ macro_rules! expand_round {
         t1 = _mm_xor_si128(t1, t3);
         t1 = _mm_xor_si128(t1, t2);
 
-        _mm_store_si128($enc_keys.as_mut_ptr().offset($pos), t1);
+        $enc_keys[$pos] = t1.into();
         let t1 = if $pos != 10 { _mm_aesimc_si128(t1) } else { t1 };
-        _mm_store_si128($dec_keys.as_mut_ptr().offset($pos), t1);
+        $dec_keys[$pos] = t1.into();
```

Comment:
Do you need this `into()`? IIUC `t1` has type `__m128i`, same as the array elements.

Reply:
Weird, wonder why clippy didn't catch that...

Anyway, fixed in a802938